src/main/java/com/auth0/AuthorizeUrl.java

```diff
@@ -13,23 +17,55 @@
 public class AuthorizeUrl {
 
     private static final String SCOPE_OPENID = "openid";
-    private final HttpServletRequest request;
+
+    private HttpServletResponse response;
+    private HttpServletRequest request;
     private final AuthorizeUrlBuilder builder;
+    private final String responseType;
+    private boolean legacySameSiteCookie = true;
+    private String nonce;
+    private String state;
+
     private boolean used;
 
     /**
+     * Creates a new instance that can be used to build an Auth0 Authorization URL.
+     *
+     * While this class can be used directly, it's recommended that you use {@link AuthenticationController#buildAuthorizeUrl(HttpServletRequest, HttpServletResponse, String)}
```

Comment:
Ah, yes it is package-private. Updated the comment to remove the implication that it can be used publicly, and added comments about storage and SameSite.